public/svg/EyeOpen.tsx

```diff
@@ -0,0 +1,25 @@
+const EyeOpen = () => {
+  return (
+    <svg
+      className="size-[24px]"
+      xmlns="http://www.w3.org/2000/svg"
+      fill="none"
+      viewBox="0 0 24 24"
+      strokeWidth={2}
+      stroke="currentColor"
+    >
+      <path
+        strokeLinecap="round"
```

Comment:
HNG don taya you bah?

Reply:
Lucide is recommended

Reply:
I shouldn't use svg?
Was trying to follow the design

Reply:
Okay I'll use lucide